lib/api/services/wallet_service.dart

```diff
@@ -1,210 +1,149 @@
+import 'dart:convert';
+import 'dart:typed_data';
+
+import 'package:arbor/api/responses/record_response.dart';
+import 'package:arbor/api/responses/records_response.dart';
 import 'package:arbor/api/responses/transaction_response.dart';
-import 'package:arbor/api/responses/wallet_address_response.dart';
+import 'package:arbor/bls/ec.dart';
+import 'package:arbor/bls/private_key.dart';
+import 'package:arbor/bls/schemes.dart';
+import 'package:arbor/clvm/program.dart';
+import 'package:arbor/core/utils/puzzles.dart';
 import 'package:arbor/models/transaction.dart';
-
-import '/models/models.dart';
+import 'package:bech32m/bech32m.dart';
+import 'package:bip39/bip39.dart';
+import 'package:hex/hex.dart';
 import 'package:http/http.dart' as http;
-import 'dart:convert';
 
+import '/models/models.dart';
 import '../responses.dart';
 import 'api_service.dart';
 
 class WalletService extends ApiService {
-  WalletService({this.baseURL = ApiService.baseURL});
+  @override
+  String baseURL = 'https://dev.arborwallet.com/api';
+
+  Future<Wallet> createNewWallet() async {
+    var mnemonic = generateMnemonic();
```

Comment:
@Rigidity - Can any of these functions ( generateMnemonic, mnemonicToSeed...etc) fail or return and unexpected value? Do we need to wrap each method call into a try/catch so we can propagate relevant errors to the user?

Reply:
Any computation whose result is indeterminate at compile-time and depends on a number of other factors should be put in try-catch blocks. We would need to do that to this and the others

Reply:
There's a lot happening here that we can't just leave to chance